tests/test_context.py

```diff
@@ -111,3 +111,18 @@ def test_ctx_leafref_extended(self):
         with Context(YANG_DIR, leafref_extended=True) as ctx:
             mod = ctx.load_module("yolo-leafref-extended")
             self.assertIsInstance(mod, Module)
+
+    def test_ctx_disable_searchdirs_and_clb(self):
+        def get_module_clb(mod_name, *_):
+            YOLO_NODETYPES_MOD_PATH = os.path.join(YANG_DIR, "yolo/yolo-nodetypes.yang")
+            self.assertEqual(mod_name, "yolo-nodetypes")
```

Comment:
Let's have two tests, those features are not that related.

Reply:
Ok, done within #104